packages/chat-headless/tests/chatheadless.test.ts

```diff
@@ -343,3 +346,58 @@ it("restartConversation works as expected", () => {
   };
   expect(chatHeadless.state).toEqual(expectedState);
 });
+
+describe("loadSessionState works as expected", () => {
+  const expectedState = {
+    conversationId: "dummy-id",
+    messages: [
+      {
+        text: "How can I help you?",
+        source: MessageSource.BOT,
+        timestamp: "2023-05-15T17:39:58.019Z",
+      },
+    ],
+    notes: {
+      currentGoal: "GOAL",
+    },
+    isLoading: true,
+  };
+  it("loads valid state from session storage", () => {
+    sessionStorage.setItem(
+      STATE_SESSION_STORAGE_KEY,
+      JSON.stringify(expectedState)
+    );
+    const chatHeadless = new ChatHeadless(config);
+    chatHeadless.setState({
```

Comment:
this line is not needed. The state from session storage should already be set when instantiate ChatHeadless so we can just assess from that!

Reply:
Done, I added this because of the meta field, but I changed it to just assert that meta is an empty object.